code/09-conclusion-challenge-2/Makefile

```diff
@@ -10,7 +10,7 @@ ZIPF_ARCHIVE=$(ZIPF_DIR).tar.gz
 .PHONY : all
 all : $(ZIPF_ARCHIVE)
 
-$(ZIPF_ARCHIVE) : $(ZIPF_DIR)
+$(ZIPF_ARCHIVE) : $(ZIPF_DIR) $(PNG_FILES) results.txt
```

Comment:
Hi Brandon,
Thanks for spotting this oversight. The behaviour should change, but I don't think your solution is the best way to do it. The dependencies are correct as they are - it's just that the `zipf_analysis` folder time stamp doesn't change when the updated files are copied into it. Hence it only works correctly the first time it is run. I propose leaving the dependencies as they are, but adding `touch $@` to the `$(ZIPF_DIR)` rule to update the time stamp.

```
$(ZIPF_DIR): Makefile results.txt \
             $(DAT_FILES) $(PNG_FILES) \
             $(COUNT_SRC) $(PLOT_SRC) $(ZIPF_SRC)
	mkdir -p $@
	cp $^ $@
	touch $@
```
Thanks